components/button/index.stories.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import ButtonInput from './button';
+import { Story } from '@storybook/react';
+
+type ButtonProps = {
+  intext: string;
+  disabled: boolean;
+  onClick: () => void;
+};
+
+const Template: Story<ButtonProps> = (args) => <ButtonInput {...args} />;
```

Comment:
Rename file to just stories.tsx.

Here and other places as well.

Reply:
done